streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -3778,17 +3777,13 @@ public Map<TopicPartition, Long> purgeableOffsets() {
 
         purgableOffsets.put(t1p1, 17L);
         taskManager.maybePurgeCommittedRecords();
-
-        verify(adminClient);
```

Comment:
This line is removed, and I wonder whether we should add a similar assertion for the `adminClient` mock after each of these two calls: https://github.com/apache/kafka/blob/ff6b6dd601d9a6156dba249a8b52201a34eaca13/streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java#L3775-L3779

Reply:
Heya, thank you for the review! I am not quite certain I follow what you mean. The reason this is removed here and elsewhere is that Mockito verifies internally that all `when` statements are called. EasyMock does not do this by default so it requires an explicit verification.

Reply:
You are right; I didn't notice that we are using `STRICT_STUBS` strictness: https://github.com/apache/kafka/blob/ff6b6dd601d9a6156dba249a8b52201a34eaca13/streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java#L208

Thanks for the reply!